lib/rbi/rewriters/replace_attributes_with_methods.rb

```diff
@@ -0,0 +1,146 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RBI
+  module TempHelpers
+    extend T::Sig
+  end
+
+  module Rewriters
+    class ReplaceAttributesWithMethods < Visitor
+      extend T::Sig
+      include TempHelpers
```

Comment:
```suggestion

      include TempHelpers
```

Reply:
Ah, these were indeed meant to be temporary while I refactored stuff around. They'll be gone shortly.